ci/benchmark_ai.sh

```diff
@@ -0,0 +1,149 @@
+    wget http://jmeter-plugins.org/downloads/file/JMeterPlugins-WebDriver-1.4.0.zip
+    wget http://jmeter-plugins.org/files/packages/jpgc-synthesis-2.1.zip
+    unzip -n jpgc-synthesis-2.1.zip -d /tmp
+    unzip -n JMeterPlugins-Standard-1.4.0.zip -d $HOME/apache-jmeter-4.0
+    unzip -n JMeterPlugins-Extras-1.4.0.zip -d $HOME/apache-jmeter-4.0
+    unzip -n JMeterPlugins-ExtrasLibs-1.4.0.zip -d $HOME/apache-jmeter-4.0
+    unzip -n JMeterPlugins-WebDriver-1.4.0.zip -d $HOME/apache-jmeter-4.0
+    cp ./lib/jmeter-plugins-cmn-jmeter-0.4.jar $HOME/apache-jmeter-4.0/lib/
+    cp ./lib/ext/* $HOME/apache-jmeter-4.0/lib/ext
+fi
+# end of jmeter installation
+
+echo "setting up conf"
+#Change conf files for running resnet
+cd /
+if [ -d "/mxnet-model-server" ]; then
+    git clone https://github.com/awslabs/mxnet-model-server.git
+fi
+sed -i -e 's/squeezenet=https:\/\/s3.amazonaws.com\/model-server\/models\/squeezenet_v1.1\/squeezenet_v1.1.model/resnet-18=https:\/\/s3.amazonaws.com\/model-server\/models\/resnet-18\/resnet-18.model /g'\
+ /mxnet-model-server/docker/mms_app_cpu.conf
+sed -i -e 's/squeezenet=https:\/\/s3.amazonaws.com\/model-server\/models\/squeezenet_v1.1\/squeezenet_v1.1.model/resnet-18=https:\/\/s3.amazonaws.com\/model-server\/models\/resnet-18\/resnet-18.model /g'\
+ /mxnet-model-server/docker/mms_app_gpu.conf
+sed -i -e 's/\/usr\/local\/Cellar\/jmeter\/3.3\/libexec\/lib\/ext\/CMDRunner.jar/\/home\/ubuntu\/apache-jmeter-4.0\/lib\/ext\/CMDRunner.jar /g' /mxnet-model-server/load-test/run_load_test.sh
+
+
+file="$HOME/log_results"
+
+if [ -f $file ] ; then
+    rm $file
+fi
+
+#start the container test
+for workers in 50 100
+do
+    for requests in 25 50 100
+    do
+        echo "*************************************** Workers $workers and reqs $requests   ****************************************************"
+        if [ "$use_gpu" == "0" ]; then
+            docker run  --name mms -v /mxnet-model-server:/mxnet-model-server  -itd -p 80:8080 awsdeeplearningteam/mms_cpu
+            docker exec mms pip install -U -e /mxnet-model-server/.
+            docker exec mms mxnet-model-server start --mms-config /mxnet-model-server/docker/mms_app_cpu.conf >& /dev/null &
+            sleep 30
+        elif [ "$use_gpu" == "1" ]; then
+            nvidia-docker run --name mms -v /mxnet-model-server:/mxnet-model-server -itd -p 80:8080 awsdeeplearningteam/mms_gpu
+            nvidia-docker exec mms pip install -U -e /mxnet-model-server/.
+            nvidia-docker exec mms pip uninstall --yes mxnet-cu90mkl
+            nvidia-docker exec mms pip uninstall --yes mxnet
+            nvidia-docker exec mms pip install mxnet-cu90mkl
+            nvidia-docker exec mms mxnet-model-server start --mms-config /mxnet-model-server/docker/mms_app_gpu.conf >& /dev/null &
+            sleep 90
+        fi
+
+        cd /mxnet-model-server/load-test
+        export JMETER_HOME=$HOME/apache-jmeter-4.0
```

Comment:
Do these exports change for every run of the test? If not, we should do this at the beginning of the test